website_portal_medical_prescription_order_line/controllers/main.py

```diff
@@ -28,6 +28,21 @@ def my_medical(self, **kw):
             ('patient_id.parent_id', 'child_of', [partner_id.id]),
         ])
 
+        url = str(request.httprequest)
```

Comment:
I'm unclear on what URI parameters are even being extracted here. What was the exact issue with getlist?